[2023-02-04 06:38:59,278] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T06:38:55.385094+00:00 [queued]>
[2023-02-04 06:38:59,286] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T06:38:55.385094+00:00 [queued]>
[2023-02-04 06:38:59,287] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 06:38:59,288] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 06:38:59,288] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 06:38:59,302] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2023-02-04 06:38:55.385094+00:00
[2023-02-04 06:38:59,307] {standard_task_runner.py:52} INFO - Started process 3247 to run task
[2023-02-04 06:38:59,310] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2023-02-04T06:38:55.385094+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag_test.py', '--cfg-path', '/tmp/tmpz1z6vcxi', '--error-file', '/tmp/tmpapsia5yk']
[2023-02-04 06:38:59,310] {standard_task_runner.py:80} INFO - Job 54: Subtask insert_data
[2023-02-04 06:38:59,371] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2023-02-04T06:38:55.385094+00:00 [running]> on host 39df186e1726
[2023-02-04 06:38:59,463] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T06:38:55.385094+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-04T06:38:55.385094+00:00
[2023-02-04 06:38:59,475] {logging_mixin.py:115} INFO - rate_date:  2023-02-04
[2023-02-04 06:38:59,476] {logging_mixin.py:115} INFO - value_:  23510.60869599999932
[2023-02-04 06:38:59,485] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-02-04 06:38:59,502] {logging_mixin.py:115} INFO - [(8,)]
[2023-02-04 06:38:59,503] {python.py:173} INFO - Done. Returned value was: None
[2023-02-04 06:38:59,516] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20230204T063855, start_date=20230204T063859, end_date=20230204T063859
[2023-02-04 06:38:59,563] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 06:38:59,600] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
